compose.yml

```diff
@@ -96,7 +96,7 @@ services:
 
   temporal-seed:
     container_name: neosync-temporal-seed
-    image: temporalio/admin-tools:latest
+    image: temporalio/admin-tools:1.23
```

Comment:
Hm, the temporal version we have set in the .env is `TEMPORAL_VERSION=1.22.6`
Should this be 1.22 then? Also open to bumping the temporal compose to the latest.

Reply:
1.22 works as well - I just tried to find a middle ground but didn't realize that `.env` pinned this version

Reply:
I see, ok. 
In the temporal compseo script I realized we actually have the admin tools defined, I commented them out a long time ago.

```
  # temporal-admin-tools:
  #   container_name: temporal-admin-tools
  #   depends_on:
  #     - temporal
  #   environment:
  #     - TEMPORAL_ADDRESS=temporal:7233
  #     - TEMPORAL_CLI_ADDRESS=temporal:7233
  #   image: temporalio/admin-tools:${TEMPORAL_VERSION}
  #   networks:
  #     - temporal-network
  #   stdin_open: true
  #   tty: true
```
Perhaps longterm I should go back to using that, and then interact with the container externally instead of doing what we are currently doing.
Either way, that is outside of the scope of this issue.

If the 1.23 image is working fine for you, let's go with it for now. Can figure out the better optimization later.

Reply:
Good to know 1.24 has some issues. I will create a ticket and look into that.

Reply:
FYI, created two followup issues: https://github.com/nucleuscloud/neosync/issues/2133 and https://github.com/nucleuscloud/neosync/issues/2134